capacitor.config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "appId": "org.leapchat",
+  "appName": "LeapChat",
+  "webDir": "build",
+  "bundledWebRuntime": false,
+  "plugins": {
+    "CapacitorHttp": {
+      "enabled": true
+    }
+  },
+  "server": {
+    "cleartext": "true",
+    "allowNavigation": [
+      "192.168.1.247:8080/*",
+      "localhost:8080/*",
+      "leapchat.org/*"
```

Comment:
🙏 📿

Reply:
Sweet! Should probably add `www.leapchat.org/*`, I'd think?

Reply:
And 10.0.2.2 -- https://developer.android.com/studio/run/emulator-networking

Reply:
`"10.0.2.2:8080/*"`, I suppose